src/frequenz/client/dispatch/__main__.py

```diff
@@ -239,9 +239,9 @@ def validate_reccurance(ctx: click.Context, param: click.Parameter, value: Any)
     required=True,
     type=str,
 )
-@click.argument("start-time", required=True, type=FuzzyDateTime())
-@click.argument("duration", required=True, type=FuzzyTimeDelta())
 @click.argument("selector", required=True, type=SelectorParamType())
+@click.argument("start-time", required=True, type=FuzzyDateTime())
+@click.argument("duration", required=False, type=FuzzyTimeDelta())
```

Comment:
In the second commit there are quite a few reordering changes that don't seem related, here and also in many imports or string lists.

Reply:
Yeah, I blame black.

Reply:
Ah no, the reordering here is very much related. `duration` became an optional positional parameter, so it had to move to the end

Reply:
Really, if black is reordering that it should be a nasty bug, decorators are not commutable, if you change the order the results could be completely different!